entrypoint.py

```diff
@@ -120,4 +120,4 @@ class EbooksEntryPoint(MediumEntryPoint):
 
 class AudiobooksEntryPoint(MediumEntryPoint):
     INTERNAL_NAME = "Audio"
-EntryPoint.register(AudiobooksEntryPoint, "Audiobooks")
+EntryPoint.register(AudiobooksEntryPoint, "Audio")
```

Comment:
This change matches our UI design for entry point navigation, where horizontal space is at a premium.